
package publicar;

import javax.xml.ws.WebFault;


/**
 * This class was generated by the JAX-WS RI.
 * JAX-WS RI 2.2.9-b130926.1035
 * Generated source version: 2.2
 * 
 */
@WebFault(name = "UsuarioAgregarYaExisteException", targetNamespace = "http://publicar/")
public class UsuarioAgregarYaExisteException_Exception
    extends Exception
{

    /**
     * Java type that goes as soapenv:Fault detail element.
     * 
     */
    private UsuarioAgregarYaExisteException faultInfo;

    /**
     * 
     * @param faultInfo
     * @param message
     */
    public UsuarioAgregarYaExisteException_Exception(String message, UsuarioAgregarYaExisteException faultInfo) {
        super(message);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @param faultInfo
     * @param cause
     * @param message
     */
    public UsuarioAgregarYaExisteException_Exception(String message, UsuarioAgregarYaExisteException faultInfo, Throwable cause) {
        super(message, cause);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @return
     *     returns fault bean: publicar.UsuarioAgregarYaExisteException
     */
    public UsuarioAgregarYaExisteException getFaultInfo() {
        return faultInfo;
    }

}
